services/authentication/src/main/java/control/CredentialsChecker.java

```diff
@@ -0,0 +1,21 @@
+package control;
+
+/**
+ * Created by ste on 12/07/2017.
+ */
+/**
+ * @author Stefano Bernagozzi
+ * This inteface models the authentication logic.
+ */
+
+public interface CredentialsChecker {
+    /**
+     * Returns the success or failure of the authentication process given the user's username and password in input.
+     * @param username Identifies the username associated to the <em>user</em>.
```

Comment:
The words after @param are the name of the parameter in the method's parameters list.